### PR TITLE
Update XbmcMetadata.cs

### PR DESCRIPTION
**Pull request by coolasice1999**
#### Database Migration
NO

#### Description
Added kodi trailer support in metadata via youtube plugin

#### Issues Fixed or Closed by this PR

* #

### CODE REVIEW

File: src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs [hunk 1]
```diff
@@ -145,6 +145,13 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
                     details.Add(imdbId);
                 }
 
+                if(movie.YouTubeTrailerId.IsNotNullOrWhiteSpace())
+                {
+                String[] tube = { "plugin://plugin.video.youtube/?action=play_video&amp;videoid=", movie.YouTubeTrailerId};
+                details.Add(new XElement("trailer", string.Join("", tube)));
+                }
+
```
Comment (galli-leo) [inline src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs]: Can you remove this blank line?

File: src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs [hunk 2]
```diff
@@ -145,6 +145,13 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
                     details.Add(imdbId);
                 }
 
+                if(movie.YouTubeTrailerId.IsNotNullOrWhiteSpace())
+                {
+                String[] tube = { "plugin://plugin.video.youtube/?action=play_video&amp;videoid=", movie.YouTubeTrailerId};
```
Comment (galli-leo) [inline src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs]: Can you fix the indentation here?

**Comment by coolasice1999** [comment]
I'll see what I can do next week.  Away from my laptop for a bit.

File: src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs [hunk 3]
```diff
@@ -145,6 +145,12 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
                     details.Add(imdbId);
                 }
 
+                if(movie.YouTubeTrailerId.IsNotNullOrWhiteSpace())
+                {
+                    String[] tube = { "plugin://plugin.video.youtube/?action=play_video&amp;videoid=", movie.YouTubeTrailerId};
+                    details.Add(new XElement("trailer", string.Join("", tube)));
```
Comment (Qstick) [inline src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs]: Why make this a list and join? Aren’t we always just combining the two strings? Should be able use string interpolation

**Comment by coolasice1999** [comment]
If you know a better way please provide.  I don't have a clue about programming and was able to figure this out the past 2 days.

**Comment by Qstick** [comment]
```
var youtubeLink = string.Format("plugin://plugin.video.youtube/?action=play_video&amp;videoid={0}", movie.YouTubeTrailerId);
details.Add(new XElement("trailer", youtubeLink));
```

Should avoid creating a list and then running a join on it for a single string.

**Pull request closed without merging**